src/StackExchange.Utils.Http/HttpBuilder.cs

```diff
@@ -28,6 +29,7 @@ public HttpBuilder(string uri, HttpSettings settings, string callerName, string
             };
             Settings = settings;
             Timeout = (settings ?? Http.DefaultSettings).DefaultTimeout;
+            BufferResponse = true;
```

Comment:
In a serialization context it's possible to skip constructors - let's initialize like `LogErrors` consistently above please :)
```suggestion
```